pkg/daemon/ceph/osd/daemon.go

```diff
@@ -66,12 +66,12 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID, lvPath string,
 		go handleTerminate(context, lvPath, volumeGroupName)
 
 		if err := context.Executor.ExecuteCommand(false, "", "vgchange", "-an", volumeGroupName); err != nil {
-			return fmt.Errorf("failed to deactivate volume group for lv %+v. Error: %+v", lvPath, err)
+			return fmt.Errorf("failed to deactivate volume group for lv %+v. %+v", lvPath, err)
 		}
+	}
 
-		if err := context.Executor.ExecuteCommand(false, "", "vgchange", "-ay", volumeGroupName); err != nil {
-			return fmt.Errorf("failed to activate volume group for lv %+v. Error: %+v", lvPath, err)
-		}
+	if err := context.Executor.ExecuteCommand(false, "", "vgchange", "-ay", volumeGroupName); err != nil {
+		return fmt.Errorf("failed to activate volume group for lv %+v. %+v", lvPath, err)
```

Comment:
Isn't `lvPath` a string? If so please use `%q` instead. Thanks.

Reply:
Fixed it now.